qa-tests-backend/src/test/groovy/VulnMgmtSACTest.groovy

```diff
@@ -55,6 +83,124 @@ class VulnMgmtSACTest extends BaseSpecification {
     }
     """
 
+    private static final GET_IMAGE_COMPONENTS_QUERY = """
+    query getComponents(\$query: String, \$pagination: Pagination)
+    {
+        results: imageComponents(query: \$query, pagination: \$pagination) {
+            ...componentFields
+            __typename
+        }
+        count: imageComponentCount(query: \$query)
+    }
+    fragment componentFields on ImageComponent {
+        name
+        version
+    }
+    """
+
+    private static final GET_NODE_COMPONENTS_QUERY = """
+    query getComponents(\$query: String, \$pagination: Pagination)
+    {
+        results: nodeComponents(query: \$query, pagination: \$pagination) {
+            ...componentFields
+            __typename
+        }
+        count: nodeComponentCount(query: \$query)
+    }
+    fragment componentFields on NodeComponent {
+        name
+        version
+    }
+    """
+
+    private static final IMAGES_TO_RESCAN = [
```

Comment:
How did you get this list?

Reply:
I went through the list of previously run test files and looked for what looked like an image name.
The reason why I did this (and left debugging stuff in the test bodies) is that at some point, the image test was failing with one extra image vulnerability when using SAC than when using scoping query `"Image:*"`. I suspect reprocessing or pruning may have left an orphaned ImageCVE behind, hence the re-scan.

Reply:
Not related to this PR but do we really need 85 images to do our E2E tests?
cc @gavin-stackrox @msugakov

Reply:
Not an answer to @janisz question, but a not for @rhybrillou . Please make sure you're aware of multi-platform enablement which soon will change images from `quay.io/rhacs-eng/qa` to `quay.io/rhacs-eng/qa-multi-arch`. https://github.com/stackrox/stackrox/pull/3461